src/tokenized-line.coffee

```diff
@@ -111,27 +111,32 @@ class TokenizedLine
 
       return maxColumn
 
-  # Calculates how many trailing spaces in this line's indentation cannot fit in a single tab.
+  # For a given `indentLevel`, calculates how many trailing spaces cannot fit in a single tab.
   #
-  # Returns a {Number} representing the odd indentation spaces in this line.
-  getOddIndentationSpaces: ->
-    oddIndentLevel = @indentLevel - Math.floor(@indentLevel)
+  # indentLevel - {Number}
+  # Returns a {Number} representing the odd indentation spaces in `indentLevel`.
+  oddIndentationSpacesForIndentLevel: (indentLevel) ->
+    oddIndentLevel = indentLevel - Math.floor(indentLevel)
     Math.round(@tabLength * oddIndentLevel)
 
-  buildSoftWrapIndentationTokens: (token) ->
-    indentTokens = [0...Math.floor(@indentLevel)].map =>
+  buildSoftWrapIndentationTokens: (token, hangingIndent) ->
```

Comment:
If you change the implementation of this method, you don't need `oddIndentationSpacesForIndentLevel` at all. 

```coffee
  buildSoftWrapIndentationTokens: (token, hangingIndent) ->
    totalIndentSpaces = (@indentLevel * @tabLength) + hangingIndent
    indentTokens = []
    while totalIndentSpaces > 0
      tokenLength = Math.min(@tabLength, totalIndentSpaces)
      indentToken = token.buildSoftWrapIndentationToken(tokenLength)
      indentTokens.push(indentToken)
      totalIndentSpaces -= tokenLength
    indentTokens
```

I tested this version locally, but wanted to avoid committing to this branch without your :+1:.